source/mapping-rules/field-customizations/field-customizations.txt

```diff
@@ -0,0 +1,74 @@
+.. _rm-field-customizations:
+
+====================
+Field Customizations
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Field customizations are optional components of your mapping rules that 
+allow you to convert your source relational column data types into a 
+customized :ref:`BSON data types <bson-types>`. 
+
+Relational Migrator assigns a default data type to your MongoDB fields 
+that most closely matches the source column data type. Field customization 
+gives you the ability to change the MongoDB field data type to fine-tune 
+your schema.
+
+Use Cases
+---------
+
+You can use field customization to:
+
+- Change a source relational datatype to a specific MongoDB BSON datatype.
+- Customize ``_id`` fields using JavaScript expressions.
+- Configure null handling behavior.
+- Rename your fields.
+
+Behavior
```

Comment:
Not sure if using field customization effects data verification, if so we should add a callout here. Can you advise @tomhollander ?